ui/src/components/InputDialog/Fields.tsx

```diff
@@ -12,7 +12,11 @@ interface RegularField {
 interface SelectionField {
   label: string;
   type: 'selection';
-  items: string[];
+  items: DropdownItemProps[];
+}
+
+export function defaultItems(items: string[]): DropdownItemProps[] {
```

Comment:
can this be 'formatDefaultDropdownItems' or something? 'defaultItems' is pretty vague

Reply:
made it so that both `string[]` and `DropdownItems[]` are allowed as `items`, and it internally converts the string array to dropdownItemProps, if needed 

that way an external function isn't necessary